src/utils/index.js

```diff
@@ -30,6 +30,13 @@ const mrjsUtils = {
     Physics,
     StringUtils,
     xr,
+
+    get appScale() {
```

Comment:
this should be a function within mr-app even if it is not object dependent - there should be a way to do a `static` function of the class in javascript for it

Reply:
I thought about this but not everything has access to `mr-app`. and I'd like to keep it that way, as it'd make things harrier.

Reply:
gotcha, then doing something in like `utils/app.js` should be fine (or whatever js you want to call it in `utils/...`) - we just shouldnt directly put a function in the index.js is all for better scalability and future testing

Reply:
Yeah that's a fair point. I'm cooking and haven't tested your PR yet. I'll make the change after finishing both of those.

Reply:
yea no rush ~